modules/tile-converter/src/i3s-converter/i3s-converter.ts

```diff
@@ -105,6 +108,8 @@ export default class I3SConverter {
     this.layers0Path = '';
     this.materialMap = new Map();
     this.materialDefinitions = [];
+    this.geometryMap = new Map();
```

Comment:
Please type the map and array if possible. I suppose this class needs a typing pass, but at least try to add types when changing things?

Reply:
I see. To make it clearer, I would recommend moving all these default initializations into the class declaration. Only initializations that depend on constructor arguments need to go into the constructor.